### PR TITLE
Enable unstable features when editing in VS Code

### PR DESCRIPTION
**Pull request by AArnott**
This improves the editing experience by 'enabling' all the code that otherwise is turned off because of the `unstable` flag.

### CODE REVIEW

**Review by nuttycom — COMMENTED** [review]
In general we don't commit editor-specific config to this respository. I'm not strongly opposed to it in this case, but maybe @str4d or another vscode user can chime in here.

**Comment by AArnott** [comment]
It took me a while to figure out how to get a good editing experience by adding this file. As it doesn't hurt any _other_ IDE to add it, IMO we should add it so that any VS Code user doesn't have to independently discover how to do it, and maintain a file like this without checking it in by accident.
Either way, it's not blocking me. It's in my own fork of the repo where I do most of my work.

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "rust-analyzer.cargo.features": ["unstable"]
```
Comment (str4d) [inline .vscode/settings.json:2]: I develop locally with:
```suggestion
    "rust-analyzer.cargo.features": "all"
```

**Review by str4d — CHANGES_REQUESTED** [review]
I'm fine with including this in the repo to help VS Code users, but we shouldn't special-case the `unstable` flag; there are other flags that are also relevant.

**Review by str4d — APPROVED** [review]
ACK (haven't tested this PR but I have the identical setting locally)

**Pull request merged**